astilectron.go

```diff
@@ -161,6 +161,11 @@ func (a *Astilectron) On(eventName string, l Listener) {
 	a.dispatcher.addListener(targetIDApp, eventName, l)
 }
 
+// OnPowerEvent implements the Listenable interface for power related events
```

Comment:
This method feels weird to me. I'd rather people use the default `(a *astilectron) On()` method to get power related events.

Therefore I would :

- remove this function
- remove the power target id and send power related events to the app target id

Reply:
Done